feinstaub/sensors/models.py

```diff
@@ -175,6 +175,10 @@ class SensorLocation(TimeStampedModel):
 
     class Meta:
         ordering = ['location', ]
+        indexes = [
+            models.Index(fields=['country', 'city']),
+            models.Index(fields=['city'], name='city_idx'),
```

Comment:
Any difference with adding [`db_index`](https://docs.djangoproject.com/en/2.2/ref/models/fields/#db-index) on the field itself?

Reply:
@kilemensi I tested this, and django did not update the migration file to create an index. Apparently, it creates if the table is nonexistent and first created.